crates/biome_parser/src/diagnostic.rs

```diff
@@ -18,8 +19,9 @@ use std::cmp::Ordering;
 ///
 /// These information **are printed in this exact order**.
 ///
-#[derive(Debug, Diagnostic, Clone)]
+#[derive(Clone, Debug, Deserialize, Diagnostic, Serialize)]
```

Comment:
These shouldn't be deserialisable/serialisable. We have the `diagnostics::serde` module for that. Can you revert the changes *inside this file*?